cmd/collapsed-kube-commit-mapper/main.go

```diff
@@ -117,7 +125,7 @@ func main() {
 		glog.Fatalf("Failed to get first-parent commit list for %s: %v", dstHead.Hash, err)
 	}
 
-	kubeCommitsToDstCommits, err := git.KubeCommitsToDstCommits(r, dstFirstParents, kFirstParents)
+	kubeCommitsToDstCommits, err := git.KubeCommitsToDstCommits(r, *repoOrg, *repoName, dstFirstParents, kFirstParents)
```

Comment:
KubeCommitToDstCommits -> UpstreamCommitToDstCommits

Reply:
or better SourceCommitToDstCommits. Wdyt?

Reply:
later sound fine to me